tests/test_failure_propagator.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier1
+
+
+@tier1
+@pytest.mark.last
```

Comment:
We already marked one test as last already: https://github.com/red-hat-storage/ocs-ci/blob/66e42efb8b150e4e3d86467a166b5db1ddd2b419/tests/manage/z_cluster/cluster_expansion/test_add_capacity.py#L88, need to change it's order to be `second_to_last`

Reply:
great catch